Gemfile

```diff
@@ -72,10 +72,9 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'rspec-collection_matchers'
   gem 'guard-rspec', require: false
-  gem 'rubocop', require: false
-
   gem 'pry'
   gem 'pry-byebug'
+  gem 'rubocop', require: false
```

Comment:
Install rubocop globally on your machine using `gem install rubocop` and don't include it here, since it's not a part of this project.